package.json

```diff
@@ -15,6 +15,7 @@
 		"build:all": "rimraf dist && npm run build:client && npm run build:server",
 		"build:client": "rimraf dist/static && webpack --config build/webpack.client.prod.conf.js --progress --hide-modules",
 		"build:server": "webpack --config build/webpack.server.conf.js --progress --hide-modules",
+		"destroyBuildCache": "rimraf node_modules/.cache",
```

Comment:
Is there any concern from using the default cache location and consequently blowing away the entire node_modules cache which is used by other tools such as babel?

Reply:
Probably no concern, but I updated it to only destroy the hard-source folder.

Reply:
Thanks! @emuvente Let's chat about this briefly in our sync tomorrow then we'll merge and monitor through the next sprint.